notimetowait2.md

```diff
@@ -44,6 +46,8 @@ Thursday, November 9, 2017 and Friday, November 10, 2017
 Opening: 09:00  
 Closing: 17:00 (sharp)  
 
+The 2 day symposium includes drinks and (cold) lunch on site, and a social dinner is planned on Thursday
```

Comment:
OK, maybe the Thursday here it related to the dinner mentioned on line 17, but they are far apart. Is `(cold)` necessary?

Reply:
'cold' is not necessary, only for debate. I can remove it.
This is sae dinner, only technical part (the date). Suggestion?